### ISSUE TITLE
Error: Cannot find module 'worker-loader?inline&fallback=false!./worker.js'

### ISSUE THREAD

**Issue by irudoy** [zerobias/telegram-mtproto#94]
```
{ Error: Cannot find module 'worker-loader?inline&fallback=false!./worker.js'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at getWorker (/.../node_modules/telegram-mtproto/lib/co-worker/web-worker.js:88:22)
    at Webworker.initWorker (/.../node_modules/telegram-mtproto/lib/co-worker/web-worker.js:52:19)
    at new Webworker (/.../node_modules/telegram-mtproto/lib/co-worker/web-worker.js:28:10)
    at Function.of (/.../node_modules/telegram-mtproto/lib/co-worker/web-worker.js:21:12)
    at getCrypto (/.../node_modules/telegram-mtproto/lib/co-worker/index.js:27:43)
    at Object.<anonymous> (/.../node_modules/telegram-mtproto/lib/config-provider/facade.js:229:34)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/.../node_modules/telegram-mtproto/lib/config-provider/index.js:23:15)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18) code: 'MODULE_NOT_FOUND' }
Crypto worker registered
```

**Comment by goodmind** [comment]
Possible duplicate of https://github.com/zerobias/telegram-mtproto/issues/85

**Comment by zerobias** [comment]
Fixed in `3.2.10`
